.baseline/checkstyle/checkstyle.xml

```diff
@@ -307,7 +307,7 @@
         </module>
         <module name="RegexpSinglelineJava">
             <property name="format" value="\bIOUtils\.toString\("/>
-            <property name="message" value="Prefer Guava''s {CharStreams,Files,Resources}.toString to avoid charset/stream closing issues."/>
+            <property name="message" value="Prefer Guava''s [CharStreams,Files,Resources].toString to avoid charset/stream closing issues."/>
```

Comment:
@uschi2000 needed to change this otherwise checkstyle throws:
```
Caused by: java.lang.IllegalArgumentException: can't parse argument number: CharStreams
        at com.puppycrawl.tools.checkstyle.api.LocalizedMessage.getMessage(LocalizedMessage.java:265)
        at com.puppycrawl.tools.checkstyle.api.LocalizedMessage.compareTo(LocalizedMessage.java:377)
        at com.puppycrawl.tools.checkstyle.api.LocalizedMessage.compareTo(LocalizedMessage.java:49)
        at com.puppycrawl.tools.checkstyle.api.LocalizedMessages.add(LocalizedMessages.java:53)
        at com.puppycrawl.tools.checkstyle.api.Check.log(Check.java:239)
        at com.puppycrawl.tools.checkstyle.checks.regexp.SinglelineDetector.checkLine(SinglelineDetector.java:114)
        at com.puppycrawl.tools.checkstyle.checks.regexp.SinglelineDetector.processLines(SinglelineDetector.java:52)
        at com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck.beginTree(RegexpSinglelineJavaCheck.java:73)
        at com.puppycrawl.tools.checkstyle.TreeWalker.notifyBegin(TreeWalker.java:354)
        at com.puppycrawl.tools.checkstyle.TreeWalker.walk(TreeWalker.java:326)
        at com.puppycrawl.tools.checkstyle.TreeWalker.processFiltered(TreeWalker.java:203)
        at com.puppycrawl.tools.checkstyle.api.AbstractFileSetCheck.process(AbstractFileSetCheck.java:79)
        at com.puppycrawl.tools.checkstyle.Checker.process(Checker.java:265)
        at com.puppycrawl.tools.checkstyle.ant.CheckstyleAntTask.processFiles(CheckstyleAntTask.java:351)
        at com.puppycrawl.tools.checkstyle.ant.CheckstyleAntTask.realExecute(CheckstyleAntTask.java:320)
        at com.puppycrawl.tools.checkstyle.ant.CheckstyleAntTask.execute(CheckstyleAntTask.java:292)
        at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
        ... 88 more
Caused by: java.lang.NumberFormatException: For input string: "CharStreams"
        ... 106 more
```